src/Engine/Util/Util.cpp

```diff
@@ -11,10 +12,12 @@ float util::inverse_sqrt(float magnitude) {
     return magnitude * (1.5f - magnitude_half * magnitude * magnitude);
 }
 
-double angle_from_vertical_vector_to(double from_x, double from_y, double to_x, double to_y) {
+double util::angle_from_vertical_vector_to(double from_x, double from_y, double to_x, double to_y) {
     double delta_x = to_x - from_x;
     double delta_y = from_y - to_y;
 
     double theta_radians = atan2(delta_y, delta_x);
```

Comment:
I'm doing some math wrong here @enochtsang . Plz halp. I'll look into it asap, but busy most of tmr.